packages/cli/src/util.ts

```diff
@@ -200,9 +200,7 @@ export function timestampForFilenames(ts: ZonedDateTime): string {
  * - use what was provided via --region on cmdline
  * - error out if --region was not provided
  */
-export async function awsGetClusterRegionWithCmdlineFallback(): Promise<
-  string
-> {
+export async function awsGetClusterRegionWithCmdlineFallback(): Promise<string> {
```

Comment:
Sorry for the noise, but my editor did this automatically.  Am I missing a linewrap setting or something?